packages/core/src/components/datetime-picker/datetime-picker.tsx

```diff
@@ -232,7 +232,6 @@ export class DateTimePicker {
             showHour={this.showHour}
             showMinutes={this.showMinutes}
             showSeconds={this.showSeconds}
-            showTimeReference={this.showTimeReference}
```

Comment:
Looks like a breaking change to me. We should avoid doing breaking changes within this PR.

Reply:
> Looks like a breaking change to me. We should avoid doing breaking changes within this PR.

resolved